components/match2/wikis/teamfortress/match_legacy.lua

```diff
@@ -0,0 +1,130 @@
+---
+-- @Liquipedia
+-- wiki=teamfortress
+-- page=Module:Match/Legacy
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local MatchLegacy = {}
+
+local Json = require('Module:Json')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
+
+function MatchLegacy.storeMatch(match2, options)
+	local match = MatchLegacy._convertParameters(match2)
+
+	if options.storeMatch1 then
+		match.games = MatchLegacy.storeGames(match, match2)
+
+		return mw.ext.LiquipediaDB.lpdb_match(
+			'legacymatch_' .. match2.match2id,
+			match
+		)
+	end
+end
+
+function MatchLegacy.storeGames(match, match2)
+	local games = ''
+	for gameIndex, game2 in ipairs(match2.match2games or {}) do
+		local game = Table.deepCopy(game2)
+		-- Extradata
+		game.extradata = {}
+		game.extradata.gamenumber = gameIndex
+		-- Other stuff
+		game.opponent1 = match.opponent1
+		game.opponent2 = match.opponent2
+		game.opponent1flag = match.opponent1flag
+		game.opponent2flag = match.opponent2flag
+		game.date = match.date
+		local scores = game2.scores or {}
+		if type(scores) == 'string' then
+			scores = Json.parse(scores)
+		end
+		game.opponent1score = scores[1] or 0
+		game.opponent2score = scores[2] or 0
+		local res = mw.ext.LiquipediaDB.lpdb_game(
+			'legacygame_' .. match2.match2id .. gameIndex,
+			Json.stringifySubTables(game)
+		)
+		games = games .. res
+	end
+	return games
+end
```

Comment:
TF's match1 doesn't store game1 objects

Reply:
what should I delete here, all the lines that starts with `game.`?

Reply:
You can remove the entire function, just make sure to update so you don't call it as well